src/tachyonsClasses.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
If you want to reuse you tachyons classes, I would use a handlebars partial.

I think this is fine for now as styling will go away. Could we move this file out of root  though. `src/views` or something